packages/@react-spectrum/s2/stories/utils.tsx

```diff
@@ -23,8 +24,38 @@ function getBackgroundColor(staticColor: StaticColor) {
   }
   return undefined;
 }
+
 export function StaticColorProvider(props: {children: ReactNode, staticColor?: StaticColor}) {
-  return <div style={{padding: 8, background: getBackgroundColor(props.staticColor)}}>{props.children}</div>;
+  let [autoBg, setAutoBg] = useState('#5131c4');
+  return (
+    <>
+      <div
+        style={{
+          padding: 8,
+          // @ts-ignore
+          '--s2-container-bg': props.staticColor === 'auto' ? autoBg : undefined,
+          background: props.staticColor === 'auto' ? autoBg : getBackgroundColor(props.staticColor)
```

Comment:
Will we be documenting setting `--s2-container-bg` in the future (aka if a user has a custom container and wants custom background + `staticColor: auto` support) or are S2 users expected to mainly use the preset ones in Provider aka background layer color presets?

Reply:
Yeah we can document that. I was also thinking maybe we'd create a `container` macro for people to use that could automatically set it.